README.md

```diff
@@ -222,3 +223,12 @@ Hoping to keep this list updated as much as possible, any new links through PRs
 - [So You wanna use Monorepos and Micro Frontends](https://realtalkjavascript.simplecast.com/episodes/episode-99-so-you-wanna-use-monorepos-and-micro-frontends-in-your-enterprise-architecture-with-manfred-steyer) - Manfred Steyer in Real Talk JavaScript.
 - [What's so cool about Micro frontends](https://www.thoughtworks.com/podcasts/micro-frontends) - Thoughtworks Technology Podcasts.
 - [Micro-Front Ends with Natlia Venditto](https://webrush.io/episodes/episode-113-micro-front-ends-with-natlia-venditto) - Web Rush Podcast Episode 113 about Micro Front-ends.
+
+## Who Uses
+
+### Sendo
```

Comment:
We can just have the name of the company and link to their website, I don't think having a big description is necessary

Reply:
@rajasegar I updated PR, thanks for review ;)